unity-renderer/Assets/Scripts/MainScripts/DCL/CRDTProtocol/Deserializer/CRDTDeserializer.cs

```diff
@@ -0,0 +1,31 @@
+using KernelCommunication;
+
+namespace DCL.CRDT
+{
+    // Deserialize CRDT binary messages (BigEndian)
```

Comment:
Is the endianness at this point relevant? I think it should depend on the implementation of the IBinaryReader.

Reply:
🤷‍♂️ I don't know. I imagine that if someone has to modify something regarding CRDT messages they will probably land on this scripts before reaching the `IBinaryReader` and I think that knowing the endianness of the message could be relevant. So I left that comment there just in case :P